### ISSUE TITLE
Add database client to PyPi

### ISSUE THREAD

**Issue by andybaumgar** [nycmeshnet/nycmesh-support-bot#69]

**Comment by Andrew-Dickinson** [comment]
Help me understand this a bit more. You want to publish a copy of the Mesh spreadsheet client to PyPi? Is that for another project to use? IMO if it's going to get reused in multiple places it should probably have its own repo so you can do git submodules instead

**Comment by andybaumgar** [comment]
Yep, I'm using it for my analysis repo.  Brian asked me to share with  Marco, and I figured not making him clone multiple repos would be a nicer developer experience.

**Comment by Andrew-Dickinson** [comment]
Yeah, makes sense. I think we should still put it in it's own repo though,
feels weird to publish a subcomponent of this repo to PyPi

**Comment by andybaumgar** [comment]
Could be, but I'd rather not have to update the PyPi every time I make an edit for Supportbot.  The analysis script is just a extra thing.

**Comment by Andrew-Dickinson** [comment]
If you're going to upload it to PyPi, we should probably keep it up to
date? Otherwise just send Marco a zip

**Comment by andybaumgar** [comment]
That's a lot more steps.  I don't see any problem with it being out of date on PyPi.  Zip seems much more messy.

**Comment by andybaumgar** [comment]
I can put it in it's own repo, but that seems unproductive.  It will go away with the Mesh API anyways.

**Comment by andybaumgar** [comment]
I can keep the PyPi up to date if it makes you happy.  I'd rather not waste time moving it to it's own repo though.  There are lots of other issues I could be working on instead.

**Comment by Andrew-Dickinson** [comment]
Yeah don't worry about keeping it up-to-date I guess. I still think you should just send Marco a ZIP, but I guess uploading the ZIP to PyPi isn't that much more work

**Comment by andybaumgar** [comment]
There is actually an official package that does all the uploading.  You run `twine upload dist/*` and it prompts for credentials and uploads.  Most of the work was just setting up the extra package files.
